### PR TITLE
warehouse image and orchestration

### PR DESCRIPTION
**Pull request by atvaccaro**
## Description
Build and push warehouse image on pushes on main. Adds a cron action to execute this image daily, as well as when manually triggered.

## Issues
n/a

## Testing
CI must pass; we tested that the execute action works.

### CODE REVIEW

File: .github/workflows/execute-warehouse-image.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  dbt-run:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jarvusinnovations/transit-data-analytics-demo/warehouse:latest
+    steps:
+      - run: |
+          echo "$BOT_GCP_SERVICE_ACCOUNT" > /tmp/keyfile
+          BIGQUERY_SERVICE_ACCOUNT=/tmp/keyfile dbt run-operation stage_external_sources --profiles-dir . --target prod_service_account
```
Comment (lauriemerrell) [inline .github/workflows/execute-warehouse-image.yml:16]: add full refresh to this command?

**Review by lauriemerrell — APPROVED** [review]
approving assuming you will fix the error with sqlfluff, need to specify bq dialect. I also recommend turning off the automatic fixing because it is very aggressive and will change SQL in semantically meaningful ways

**Comment by atvaccaro** [comment]
Ok I think this should pass; I'm only using the `sqlfluff-lint` hook so I don't think it will auto-fix unless you're referring to some other config

**Comment by atvaccaro** [comment]
This hung so I am going to merge so we can test the manual dispatch

**Pull request merged**